binary_search/BinarySearch.java

```diff
@@ -0,0 +1,34 @@
+public class BinarySearch {
+
+    static int binarySearch(int[] arr, int arrSize, int searchElement) {
```

Comment:
It fails because there is no Javadoc comment for the function. You may ignore it as of now.